impeller/aiks/canvas.cc

```diff
@@ -23,6 +25,80 @@
 
 namespace impeller {
 
+namespace {
+
+static std::shared_ptr<Contents> CreateContentsForGeometryWithFilters(
```

Comment:
This also fixes the double application of color filters.

Reply:
Can you link an issue please.

Reply:
Filled here: https://github.com/flutter/flutter/issues/139569